web/src/lib/utils/helpers.ts

```diff
@@ -111,22 +111,41 @@ export const scrollToTop = (props?: { smooth?: boolean }): void => {
     : window.scrollTo(0, 0);
 };
 
+export const NAVBAR_WRAPPER_MOBILE_ID = "NAVBAR_WRAPPER_MOBILE_ID";
+export const NAVBAR_WRAPPER_DESKTOP_ID = "NAVBAR_WRAPPER_DESKTOP_ID";
+
+const getNavBarHeight = (): number | undefined => {
+  const navbarMobile = document.querySelector(`#${NAVBAR_WRAPPER_MOBILE_ID}`);
+  const navbarDesktop = document.querySelector(`#${NAVBAR_WRAPPER_DESKTOP_ID}`);
+
+  if (navbarDesktop && navbarMobile) {
```

Comment:
[dust] This is more for my clarity but in what case would one of these be undefined?

Reply:
If the ID was changed for some reason and now it can't find the element. It's mostly a typescript thing that required the type to not include undefined.